contracts/BandOracle.cdc

```diff
@@ -105,36 +127,77 @@ pub contract BandOracle {
                 ?? panic("Error while creating data updater capability private path")
             return dataUpdaterPrivatePath
         }
+
+        /// Removes a symbol and its quotes from the contract storage.
+        ///
+        /// @param symbol: The string representing the symbol to be removed from the contract.
+        ///
+        pub fun removeSymbol (symbol: String) {
+            BandOracle.removeSymbol(symbol: symbol)
+        }
         
-        // OracleAdmin and entitled relayers can call this method to update rates
+        /// Relayers can call this method to update rates.
+        ///
+        /// @param symbolsRates: Set of symbols and corresponding usd rates to update.
+        /// @param resolveTime: The registered time for the rates.
+        /// @param requestID: The Band Protocol request ID.
+        /// @param relayerID: The ID of the relayer carrying the update.
+        ///
         pub fun updateData (symbolsRates: {String: UInt64}, resolveTime: UInt64, 
                             requestID: UInt64, relayerID: UInt64) {
             BandOracle.updateRefData(symbolsRates: symbolsRates, resolveTime: resolveTime, 
                                     requestID: requestID, relayerID: relayerID)
         }
 
-        // OracleAdmin and entitled relayers can call this method to force update rates
+        /// Relayers can call this method to force update rates.
+        ///
+        /// @param symbolsRates: Set of symbols and corresponding usd rates to update.
+        /// @param resolveTime: The registered time for the rates.
+        /// @param requestID: The Band Protocol request ID.
+        /// @param relayerID: The ID of the relayer carrying the update.
+        ///
         pub fun forceUpdateData (symbolsRates: {String: UInt64}, resolveTime: UInt64, 
                                 requestID: UInt64, relayerID: UInt64) {
             BandOracle.forceUpdateRefData(symbolsRates: symbolsRates, resolveTime: resolveTime, 
                                     requestID: requestID, relayerID: relayerID)
         }
-        
+
+        /// Creates a fee collector, meant to be called once after contract deployment
+        /// for storing the resource on the maintainer's account.
+        ///
+        /// @return The `FeeCollector` resource
+        ///
+        pub fun createNewFeeCollector (): @FeeCollector {
+            return <- create FeeCollector()
+        }
+
     }
 
-    ///
+    /// The resource that will allow an account to make quote updates
     ///
     pub resource Relay {
```

Comment:
nit: The name `Relayer` makes sense to me here - consistent with FeeCollector and it's referred to in other parts of the contract as "relayer". I guess either works, but would prefer consistency across contract and naming conventions in their other contracts/docs